### DISCUSSION TITLE
Without a doubt the best web app I’ve ever seen

### DISCUSSION THREAD

**Discussion by sunbeam60** [aeharding/voyager#99] [General]
Honestly, I'm not sure I can tell the difference. Super smooth and blazingly fast. Just want to high five anyone that's worked on it!!

**Comment by aeharding** [comment]
Thank you so much!! 💙

**Comment by 0xCmdrKeen** [comment]
Agreed. Apollo was one of my favorite apps on iOS period. So sad when it shut down. Thankfully, Voyager fills the gap perfectly. And it runs on Android, too!
